tests/test_octodns_manager.py

```diff
@@ -286,6 +302,29 @@ def test_validate_configs(self):
                 .validate_configs()
         self.assertTrue('unknown source' in text_type(ctx.exception))
 
+        # Alias zone using an invalid source zone.
+        with self.assertRaises(ManagerException) as ctx:
+            Manager(get_config_filename('unknown-source-zone.yaml')) \
+                .validate_configs()
+        self.assertTrue('Invalid alias zone' in
+                        text_type(ctx.exception))
+
+        # Valid config file using an alias zone.
+        Manager(get_config_filename('simple-alias-zone.yaml')) \
```

Comment:
I'm loosing my mind on this one. I simply not understand of this test using `simple-alias-zone.yaml` could not trigger `continue`. 
Or maybe I'm too tired. 🤷‍♂️

Reply:
Maybe this is caused by this coverage.py bug: https://github.com/nedbat/coveragepy/issues/198#issuecomment-399705984 ?

Reply:
So yeah that's it: coverage is OK if I set the `continue` statement in a `try`/`except` block.
I've added a `pragma: no cover` comment in order to achieve 100% code coverage.